test/sumlog.jl

```diff
@@ -0,0 +1,5 @@
+@testset "sumlog" begin
+    for x in [10 .* rand(1000), repeat([nextfloat(1.0)], 1000), repeat([prevfloat(2.0)], 1000)]
```

Comment:
Can you add more tests with other number types (`Int`, `Rational`, `BigFloat`,  etc.)?

Reply:
added